components/BlogCard/BlogCard.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import Link from "next/link";
+import { IBlogCard } from "../../types/index.types";
+import styles from "./BlogCard.module.scss";
+import Icon from "../Icon/Icon";
+import contentData from "./contentData.json";
+
+type Props = {
+  blog: IBlogCard;
+};
+
+const BlogCard = () => {
+  const cardArray: any = contentData;
```

Comment:
Is there any reason we did not give a type to cardArray?

Reply:
it will be fixed later. It is like this to visualise cards for now.